spectral_connectivity/wrapper.py

```diff
@@ -0,0 +1,94 @@
+from spectral_connectivity.connectivity import Connectivity
+from spectral_connectivity.transforms import Multitaper
+import xarray as xr
+import numpy as np
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+
+def connectivity_to_xarray(m, method='coherence_magnitude', signal_names=None, squeeze=False, **kwargs):
+    """
+    calculate connectivity using `method`. Returns an xarray
+    with dimensions of ['Time', 'Frequency', 'Source', 'Target']
+    or ['Time', 'Frequency'] if squeeze=True
+
+    signal_names : iterable of strings
+        Sames of time series used to name the 'Source' and 'Target' axes of xarray
+    squeeze : bool
+        Whether to only take the first and last source and target time series. Only makes sense for
+        one pair of signals and symmetrical measures
+
+    """
+    connectivity = Connectivity.from_multitaper(m, **kwargs)
+    connectivity_mat = getattr(connectivity, method)()
+    if (m.time_series.shape[-1] == 2) and squeeze:  # Only one couple (only makes sense for symmetrical metrics)
+        logger.warning(f'Squeeze is on, but there are {m.time_series.shape[-1]} pairs!')
+        connectivity_mat = connectivity_mat[:, :, 0, -1]
+        xar = xr.DataArray(connectivity_mat,
+                           coords=[connectivity.time, connectivity.frequencies],
+                           dims=['Time', 'Frequency'])
+
+    else:  # Name the source and target axes
+        if signal_names is None:
+            signal_names = np.arange(m.time_series.shape[-1])
+        xar = xr.DataArray(connectivity_mat,
+                           coords=[connectivity.time, connectivity.frequencies, signal_names, signal_names],
+                           dims=['Time', 'Frequency', 'Source', 'Target'])
+
+    xar.name = method
+
+    for attr in dir(m):
```

Comment:
I suppose one possible concern here is that attaching everything would take up a lot of memory. I think it is fine as it is now. The dictionary can always be filtered in the future.

Reply:
Right. Should we skip `time_series` at least?

Reply:
Yes, definitely time series.